podcasts/AppDelegate+Defaults.swift

```diff
@@ -18,6 +18,7 @@ extension AppDelegate {
             Settings.setHomeFolderSortOrder(order: .dateAddedNewestToOldest)
             Settings.setMobileDataAllowed(true)
             Settings.shouldShowInitialOnboardingFlow = true
+            Settings.autoplay = true
```

Comment:
Should we disable setting this value until the feature flag is enabled?

```suggestion
            if FeatureFlag.autoplay.enabled {
                Settings.autoplay = true
            }
```